Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.5
+FROM python:3.6
```

Comment:
Shouldn't this stay on 3.5 to match the version used in production?

Reply:
reverted back to 3.5.